sputnikdao/src/lib.rs

```diff
@@ -565,7 +562,7 @@ mod tests {
         testing_env!(VMContextBuilder::new().build());
```

Comment:
You can remove all lines that setup a default context like this if you'd like. It's just replacing the default with another default. This was only needed before 4.0.

Reply:
removed